internal/bminventory/inventory.go

```diff
@@ -4117,15 +4117,23 @@ func (b *bareMetalInventory) downloadHostIgnition(ctx context.Context, clusterID
 	return fileName, respBody, contentLength, nil
 }
 
-func (b *bareMetalInventory) GetCredentials(ctx context.Context, params installer.GetCredentialsParams) middleware.Responder {
+func (b *bareMetalInventory) V2GetCredentials(ctx context.Context, params installer.V2GetCredentialsParams) middleware.Responder {
```

Comment:
please move V2 API to the inventory_v2.go file. This is were all the V2 handlers should reside from now on